### DISCUSSION TITLE
SX1262 - two way communication issue

### DISCUSSION THREAD

**Discussion by Nandhini-271** [jgromes/RadioLib#1195] [General]
I am using the Core1262-868M module (from [Waveshare](https://www.waveshare.com/core1262-868m.htm)) with an ESP32 controller. I have successfully completed long-range LoRa testing at 1km using the `RadioLib` library in a one-way communication setup, utilizing basic transmitter and receiver code. However, when attempting two-way communication using the "PingPong" example, the transmitter sends the data packet and then switches to receiver mode after 3 seconds, while the receiver remains in listening mode. Despite not altering the LoRa key parameters or antenna position, I am unable to achieve two-way communication. Could you please provide a solution for this issue?

**Comment by HeadBoffin** [comment]
Taking a cue from the very last word, I've created an issue for you!

https://github.com/jgromes/RadioLib/issues/1196
